README.md

```diff
@@ -42,17 +79,51 @@ prompt(
+The third argument is an object. It can have any of these keys:
+
+| Key                 | Description                                                                       | Type    | Default                                              |
+|---------------------|-----------------------------------------------------------------------------------|---------|------------------------------------------------------|
+| type                | Text input type: `'numeric', 'secure-text', 'phone-pad', 'email-address'`         | String  | 'default'                                            |
+| cancelable          |                                                                                   | Boolean | true                                                 |
+| defaultValue        | Default input value                                                               | String  |                                                      |
+| placeholder         |                                                                                   | String  |                                                      |
+| style               | `'default', 'shimo', 'cust'`                                                      | String  | 'default'                                            |
+| disableFullscreenUI | When in landscape mode, don't use fullscreen                                      | Boolean | false                                                |
+| highlightColor      | Color of text selection                                                           | String  | ![colorString](#colorstring) or #RRGGBB or #AARRGGBB |
+| placeholderColor    | Color of the placeholder                                                          | String  | ![colorString](#colorstring) or #RRGGBB or #AARRGGBB |
+| color               | Color of the text                                                                 | String  | ![colorString](#colorstring) or #RRGGBB or #AARRGGBB |
+| buttonColor         | Color of the buttons                                                              | String  | ![colorString](#colorstring) or #RRGGBB or #AARRGGBB |
 
- name                 | description                                 | type     | default
-:-------------------- |:------------------------------------------- | --------:|:------------------
- type                 | Text input type: `'numeric', 'secure-text', 'phone-pad', 'email-address'`  |   String | 'default'
- cancelable           |                                             |  Boolean |
- defaultValue         | Default input value                         |   String | ''
- placeholder          |                                             |   String | ''
+##### colorString
+Is one of the following: `'red', 'blue', 'green', 'black', 'white', 'gray', 'cyan', 'magenta', 'yellow', 'lightgray', 'darkgray', 'grey', 'lightgrey', 'darkgrey', 'aqua', 'fuchsia', 'lime', 'maroon', 'navy', 'olive', 'purple', 'silver', and 'teal'`
```

Comment:
You could actually support the full color format (named, rgba, etc) available in React Native by processing the color passed to the components like this:

```js
import processColor from 'react-native/Libraries/StyleSheet/processColor';

highlightColor: processColor(options.highlightColor),
placeholderColor: processColor(options.placeholderColor),
color: processColor(options.color),
buttonColor: processColor(options.buttonColor)
```

Reply:
Oh sweet! Is it possible to process color on native side? Like RCTConvert in ios?

Reply:
No, `RCTConvert` on iOS just transforms the int to a UIColor, so it can be conveniently used.
But Android native views can directly deal with Int.

Reply:
Update to support full color format! Super awesome!! would be awesome to learn how to do this on native side too.